src/api/args.ts

```diff
@@ -63,3 +63,13 @@ export type UsersQuery = Query & {
 
 export type UnitsQuery = Query & {
 };
+
+export type ObservePlateQuery = Query & {
+  scope?: string;
+  alpr?: 'openalpr' | 'sighthound' | 'platerecog';
+  method?: 'scanner';
+  filename: string;
+  latitude: number;
+  longitude: number;
+  accuracy: number;
```

Comment:
I _assume_ we're using the geolocation API.  Any reason we're not just passing the [GeolocationCoordinates](https://developer.mozilla.org/en-US/docs/Web/API/GeolocationCoordinates)?

Reply:
Cuz...

Reply:
Best reason I've heard all day.

Reply:
I'm merging, but we can revisit this later.